internal/app/repository/short_link_repository.go

```diff
@@ -0,0 +1,30 @@
+package repository
+
+import (
+	"fmt"
+	"sanbright/go_shortener/internal/app/entity"
+)
+
+type ShortLinkRepository struct {
+	Items map[string]string
```

Comment:
Мне кажется, что если мы объявили `ShortLinkEntity` то имеет смысл его тут использовать так же и для хранения ссылок. Так можно будет обосновать введение этой сущности а так же упростить связанный код - убрать лишние инициализации.
```suggestion
	Items map[string]ShortLinkEntity
```